liquid-derive/src/filter_parameters.rs

```diff
@@ -344,9 +345,10 @@ impl FromStr for FilterParameterType {
             "integer" => Ok(FilterParameterType::Integer),
             "float" => Ok(FilterParameterType::Float),
             "bool" => Ok(FilterParameterType::Bool),
+            "date_time" => Ok(FilterParameterType::DateTime),
```

Comment:
Unsure whether to do `date_time` or `datetime`.  Thoughts?

Reply:
I don't have a clear preference here.

Reply:
I'm unsure as well.  I'll go ahead and mark this resolved then.